src/LtiLibrary.NetCore/Clients/Outcomes1Client.cs

```diff
@@ -192,7 +192,10 @@ await SecuredClient.SignRequest
                                 }
                                 else
                                 {
-                                    outcomeResponse.Response = double.TryParse(imsxResponseBody.result.resultScore.textString, out var result) 
+                                    // The TC is supposed to use "en" language format, but this allows
+                                    // a little bit of misbehaving.
+                                    var cultureInfo = new CultureInfo(imsxResponseBody.result.resultScore.language??"en");
```

Comment:
> Fixed bug in Outcomes1Client that parsed score using current culture. Should always use "en".

Per your comment from the PR conversation, should this line above be simplified to:
`var cultureInfo = new CultureInfo("en")` ?

Reply:
My thinking for this was...This endpoint is called by the Tool. The onus is on the Tool to set the language to "en". If they forget, then the Consumer should assume "en".

On the other hand, if the Tool is intentionally using some other language, and they go to the trouble of specifying that language, why not use it? The use case I imagine is someone using Outcomes as an internal API between two components both owned by the same author.

Reply:
Sounds good to me!